client/my-sites/stats/summary/style.scss

```diff
@@ -0,0 +1,12 @@
+.stats-summary-view {
+	.highlight-cards-heading {
+		@media ( max-width: 1380px ) {
```

Comment:
I'm wondering about mixing `rem` and `px` units here, but you probably got it from the original styles below 660px.
If you want, you could combine those two media queries into one:
```
@media ( min-width: 661px ) and ( max-width: 1380px ) {
  margin-left: 32px;
  margin-right: 32px;
}
```
This approach will keep the default `rem` margin below 660px and apply new margins for larger screens.

Reply:
Thanks for this suggestion! I'll update and test with these changes.